examples/tags-with-hits/app.tsx

```diff
@@ -0,0 +1,399 @@
+/** @jsx h */
+import {
+  autocomplete,
+  AutocompleteComponents,
+  getAlgoliaResults,
+  getAlgoliaFacets,
+} from '@algolia/autocomplete-js';
+import {
+  AutocompleteInsightsApi,
+  createAlgoliaInsightsPlugin,
+} from '@algolia/autocomplete-plugin-algolia-insights';
+import { createTagsPlugin, Tag } from '@algolia/autocomplete-plugin-tags';
+import algoliasearch from 'algoliasearch';
+import { h, Fragment } from 'preact';
+import insightsClient from 'search-insights';
+
+import '@algolia/autocomplete-theme-classic';
+
+import { ProductHit, TagExtraData } from './types';
+import {
+  groupBy,
+  mapToAlgoliaFilters,
+  mapToAlgoliaNegativeFilters,
+} from './utils';
```

Comment:
We can use `groupBy` from Ramda, and move the Algolia functions to the root, they're not really utils.

Reply:
Handled in 8468eee and b3592e59. I've moved the two functions in `app.tsx`, we might not need separate files.